service/view_building_coordinator.hh

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2025-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+ */
+
+#pragma once
+
+#include <seastar/core/future.hh>
+#include <seastar/core/sharded.hh>
+
+#include "dht/i_partitioner_fwd.hh"
+#include "locator/host_id.hh"
+#include "schema/schema_fwd.hh"
+
+namespace service {
+
+namespace vbc {
+struct view_building_target {
+    locator::host_id host;
+    unsigned shard;
+
+    bool operator<(const view_building_target& other) const {
+        return host < other.host || (host == other.host && shard < other.shard);
+    }
```

Comment:
You can write: `std::strong_ordering operator<=>(const view_building_target&) const = default;`